src/collection-view.js

```diff
@@ -169,19 +120,21 @@ var CollectionView = Backbone.View.extend({
     return this;
   },
 
-  // An efficient rendering used for filtering. Instead of modifying
-  // the whole DOM for the collection view, we are only adding or
-  // removing the related childrenViews.
-  setFilter: function(filter, options = {}) {
-    var viewCanBeRendered = this._isRendered && !this._isDestroyed;
+  // An efficient rendering used for filtering. Instead of modifying the whole DOM for the
+  // collection view, we are only adding or removing the related childrenViews.
+  setFilter(filter, {preventRender} = {}) {
+    const viewCanBeRendered = this._isRendered && !this._isDestroyed;
```

Comment:
I know this is pre-existing behavior, but does it seem strange that if the view isn't destroyed and hasn't been rendered yet, that calling `setFilter` will not change `view.filter`?